doc/hardware/porting/board_porting.rst

```diff
@@ -361,21 +505,27 @@ named ``plank``:
 
      endif # BOARD_PLANK
 
-:file:`plank_defconfig`
+:file:`plank_defconfig` / :file:`plank_<identifier>_defconfig`
   A Kconfig fragment that is merged as-is into the final build directory
   :file:`.config` whenever an application is compiled for your board.
 
-  You should at least select your board's SOC and do any mandatory settings for
-  your system clock, console, etc. The results are architecture-specific, but
-  typically look something like this:
+  If both the common :file:`plank_defconfig` and the board identifier specific
+  :file:`plank_<identifier>_defconfig` exists then both will be used.
+  This allows you to place all configuration which is common for all SoCs, CPU
+  cores, and build variants in the base :file:`plank_defconfig` and only place
+  the adjustments specific for a given SoC or build variant in the
+  :file:`plank_<identifier>_defconfig`.
```

Comment:
Use of multiple defconfigs should be reconsidered in light of https://github.com/zephyrproject-rtos/zephyr/pull/68724.

For example, building for `qemu_cortex_a53/qemu_cortex_a53/smp` produces Kconfig warnings, because `qemu_cortex_a53_defconfig` conflicts with `qemu_cortex_a53_qemu_cortex_a53_smp_defconfig`.

Reply:
> Use of multiple defconfigs should be reconsidered in light of #68724.

Thanks for pointing to that proposal.
This PR is the doc and thus should describe the current behavior, therefore will continue the discussion related to the feature in #68724.

Reply:
Thanks for the discussion. #68724 is now closed.